### PR TITLE
Changes to Upload Script

### PR DESCRIPTION
**Pull request by sean-smith**
* Removed dependency on pcluster.utils
* Remove commented out import statements

Signed-off-by: Sean Smith <seaam@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: util/s3_factory.py [hunk 1]
```diff
@@ -118,3 +117,11 @@ def revert_object(self, s3_bucket, s3_key, version_id, dryrun=True):
...
         else:
             logging.info(f"Current version is already the requested one: {version_id}")
+
+    @staticmethod
+    def error(message, fail_on_error=True):
+        """Print an error message and Raise SystemExit exception to the stderr if fail_on_error is true."""
+        if fail_on_error:
+            sys.exit("ERROR: {0}".format(message))
+        else:
+            print("ERROR: {0}".format(message))
```
Comment (tilne) [inline util/s3_factory.py]: Could do this via `LOGGER.error`.

Comment (sean-smith) [reply util/s3_factory.py]: updated

File: util/upload-script.py [hunk 2]
```diff
@@ -164,9 +130,12 @@ def _file_type(value):
 
     args = parser.parse_args()
 
-    # get regions if no regions are specified
-    if len(args.regions) == 0:
-        args.regions.extend(_get_aws_regions(args.partition))
+    # get regions if --autodetect-regions is set
+    if args.autodetect_regions:
+        args.regions.extend(get_aws_regions(args.partition))
```
Comment (tilne) [inline util/upload-script.py]: Remove any duplicates or use a set here.

Comment (sean-smith) [reply util/upload-script.py]: Done

**Pull request merged**